Description: Remove the use of bundler while running tests
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2016-01-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,7 +1,7 @@
 $LOAD_PATH << File.expand_path('../../lib', __FILE__)
 
-require 'bundler'
-Bundler.setup
+#require 'bundler'
+#Bundler.setup
 
 require 'minitest/autorun'
 require 'minitest/mock'
